### PR TITLE
Adding a /webhooks/build

### PR DESCRIPTION
**Pull request by stjohnjohnson**
This adds a new webhook for the launcher to update the meta, status, and stopTime.

Depends on https://github.com/screwdriver-cd/data-schema/pull/20

### CODE REVIEW

File: plugins/webhooks/build.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+                        meta: request.payload.meta || {},
+                        status: request.payload.status,
+                        endTime: Date.now()
+                    }
+                }, (err) => {
+                    if (err) {
+                        return reply(boom.wrap(err));
+                    }
+
+                    return reply().code(200);
```
Comment (nkatzman) [inline plugins/webhooks/build.js]: Should this be 204? or Reply with an empty object?

Comment (jer) [inline plugins/webhooks/build.js]: 👍  for 204 No Content

**Comment by jer** [comment]
👍

**Comment by nkatzman** [comment]
👍

**Pull request merged**